src/entities/movie/api/types.ts

```diff
@@ -31,3 +31,14 @@ export interface GetMovieDto {
     trended: boolean;
     sessions: GetSessionDto;
 }
+
+export interface CreateOrUpdateMovieData {
```

Comment:
It's not a good idea to mix interfaces like this, from business logic update does not always equal to create, maybe some fields can be unchangeable
So I'd suggest to create ti separate interfaces and if you want you can create CreateMovieData and then for UpdateMoviewData extends CreateMovieData to not write the same fields

Reply:
and need to discuss types location once again, I can see we start mixing everything together :D but I don't want to block this PR